src/components/header/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import logo from '../../../public/logo.svg';
```

Comment:
🟡 paths to images could be changed to `'public/logo.svg'`

Reply:
Changed.